content/1_General/IO.mdx

```diff
@@ -487,6 +489,29 @@ int main() {
 }
 ```
 
+### Method 3: Re-defining IO
+
+The standard IO functions can also be redefined to work with file IO. The following template (which is commonly used on USACO solution pages) uses this redefinition to solve this example. After re-defining the IO functions, then you can simply use `cin` and `cout` to read and write data.
```

Comment:
State that you are redefining using https://en.cppreference.com/w/cpp/io/c/freopen